src/main/java/com/example/controller/GenerateTraitorsController.java

```diff
@@ -17,7 +18,14 @@ public class GenerateTraitorsController {
     @Autowired
     private TraitorsGenerator traitorsGenerator;
 
+    @Autowired
```

Comment:
Please rename this class accordingly, because now it handles not only traitors generating